float32 pan        #  Head pan target angle in radians. */
float32 tilt       # Head tilt target angle in radians. */
float32 speed      # /**< Maximum joint speed to reach target angles in radians/s. */
